x/precisebank/types/fractional_balances.go

```diff
@@ -0,0 +1,42 @@
+package types
+
+import (
+	fmt "fmt"
+	"strings"
+)
+
+// FractionalBalances is a slice of FractionalBalance
+type FractionalBalances []FractionalBalance
+
+// Len returns the length of the slice.
+func (fbs FractionalBalances) Len() int {
```

Comment:
This isn't used anywhere and not tested.  Remove or add an test? Tho I'm not sure I see a lot of genesis data being length checked, so do we need this?

Reply:
yeah don't need this, removed